sci-libs/cantera/cantera-2.4.0-r2.ebuild

```diff
@@ -0,0 +1,126 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+FORTRAN_NEEDED=fortran
+FORTRAN_STANDARD=90
+
+inherit desktop fortran-2 python-single-r1 scons-utils toolchain-funcs
+
+DESCRIPTION="Object-oriented tool suite for chemical kinetics, thermodynamics, and transport"
+HOMEPAGE="http://www.cantera.org"
+SRC_URI="https://github.com/Cantera/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+cti fortran pch +python test"
+
+REQUIRED_USE="
+	python? ( cti )
+	${PYTHON_REQUIRED_USE}
+	"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	python? (
+		dev-python/numpy[${PYTHON_USEDEP}]
+	)
+	<sci-libs/sundials-5.0.0:0=
+"
+
+DEPEND="
+	${RDEPEND}
+	dev-cpp/eigen
+	dev-libs/boost
```

Comment:
If this get's linked during build, it should have a subslot binder.

Reply:
The [dependencies page](https://cantera.org/compiling/dependencies.html#other-required-software) mentions that:
> Only the “header-only” portions of Boost are required. Cantera does not currently depend on any of the compiled Boost libraries. 

So Boost is not linked. All tests passed for cantera compiled against boost-1.70.